compose-builder/README.md

```diff
@@ -113,10 +139,37 @@ Options:
     ds-rest:    Generates compose file with device-rest included
     ds-snmp:    Generates compose file with device-snmp included
     ds-virtual: Generates compose file with device-virtual included
+    modbus-sim: Generates compose file with ModBus simulator included
+    asc-http:   Generates compose file with App Service HTTP Export included
+    asc-http-s: Generates compose file with App Service HTTP Export Secrets included
+    asc-mqtt:   Generates compose file with App Service MQTT Export included
+    asc-mqtt-s: Generates compose file with App Service MQTT Export Secrets included
     mqtt:       Generates compose file with services configure for MQTT Message Bus 
     ui:         Generates stand-alone compose file for EdgeX UI	
 ```
 
+```
+taf-compose [options] 
+Generates a TAF general testing compose file as specified by options and stores them in the configured TAF release folder. Compose files are named appropriatly for the options used to generate them.
+
+Options:
+    taf-secty:	  Bulds geneneral TAF testing compose file with security services
+    taf-no-secty: Bulds geneneral TAF testing compose file without security services
+    no-secty: 	  Must be use in conjunction with taf-no-secty so compose file is named appropriatly
```

Comment:
interesting. so it could not be combined into `taf-no-secty` regarding the naming?

Reply:
Actually, yes it could. Good call.